src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -155,6 +160,17 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
         return errors;
     }
 
+    public Errors validatePartialRemoveDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
```

Comment:
Need to add unit tests for this new method.

Reply:
Method should actually be made `private` I think. Testing of it would then come from the testing of the `public` method that calls it higher up the chain.

Reply:
test added method made private